doc/install_basic.txt

```diff
@@ -0,0 +1,47 @@
+.. _install_basic:
+
+Basic user install instructions
+===============================
+
+These are the 3 simple ways of obtaining Theano, and additional requirements
+will change depending on the way you choose.
+
+pip
+~~~
+    
+The recommended way to obtain the latest stable version of Theano is from PyPI using
+pip_. Use the [sudo] or [--user] option depending on your admin rights.
+
+.. code-block:: bash
+
+    [sudo] pip install Theano [--user]
+
+conda
+~~~~~
+
+say something about conda and miniconda
+
+Anaconda
+~~~~~~~~
+
+If you don't want to bother with anything, Theano can be acquired through
+Anaconda_.
+
+
+.. _pip: http://pypi.python.org/pypi/pip
+.. _anaconda: https://docs.continuum.io/
+
+.. _testing_installation:
+
+Testing your installation
```

Comment:
It is installation dependent because if you clone theano from git you would you use `theano-nose` and through pip you would use `python -c "import theano; theano.test()`